stage/dotnet/dotnet-sdk-enterprise-cloud/README.md

```diff
@@ -0,0 +1,111 @@
+# Octokit: .NET SDK
+
+[![Build and test .NET SDK](https://github.com/octokit/dotnet-sdk/actions/workflows/build.yml/badge.svg)](https://github.com/octokit/dotnet-sdk/actions/workflows/build.yml) [![CodeQL](https://github.com/octokit/dotnet-sdk/actions/workflows/github-code-scanning/codeql/badge.svg)](https://github.com/octokit/dotnet-sdk/actions/workflows/github-code-scanning/codeql) [![Publish Release to NuGet](https://github.com/octokit/dotnet-sdk/actions/workflows/publish.yml/badge.svg)](https://github.com/octokit/dotnet-sdk/actions/workflows/publish.yml)
+
+An "alpha" version of a generated .NET SDK in C# from [GitHub's OpenAPI spec](https://github.com/github/rest-api-description), built on [Kiota](https://github.com/microsoft/kiota). View on [NuGet](https://www.nuget.org/packages/GitHub.Octokit.SDK/).
+
+## How do I use it?
+
+### Installation
+
+To install the package, you can use either of the following options:
+
+- In Visual Studio, from the Package Explorer, search for `GitHub.Octokit.SDK`, or
+- Type `Install-Package GitHub.Octokit.SDK` into the Package Manager Console, or
+- Type `dotnet add ./path/to/myproject.csproj package GitHub.Octokit.SDK` in a terminal (replace `./path/to/myproject.csproj` by the path to the _*.csproj_ file you want to add the dependency)
+
+### Make your first request
+
+```csharp
+using GitHub;
```

Comment:
My guess is that this snipit might need to change if we decide to namespace differently in the event that users would want to implement dotcom and ghe* sdks in one app - like one where they can choose their platform and authenticate. Though I suppose they could alias.  I'm just trying to picture the use case, though it might not be an actual use case.  Either way these examples might need to be updated.

Reply:
Agreed! I don't believe we've made a decision on aliasing or namespacing yet; I don't have a strong preference either way. Do you?